test/RibbonThetaVault.ts

```diff
@@ -103,11 +103,11 @@ describe("RibbonThetaVault", () => {
     secondOptionStrike: 64000,
     chainlinkPricer: CHAINLINK_WBTC_PRICER,
     tokenDecimals: 6,
-    depositAmount: BigNumber.from("100000000"),
-    premium: BigNumber.from("10000000"),
+    depositAmount: BigNumber.from("100000000000"),
+    premium: BigNumber.from("10000000000"),
     premiumDiscount: BigNumber.from("997"),
     minimumSupply: BigNumber.from("10").pow("3").toString(),
```

Comment:
changed this to be consistent with eth theta vault put deposit amount for easier comparison